src/spaceandtime/apiversions.json

```diff
@@ -0,0 +1,35 @@
+{
+    "auth/token":         "v1",
+    "auth/refresh":       "v1",
+    "auth/logout":        "v1",
+    "auth/code":          "v1",
+    "auth/validtoken":    "v1",
+    "auth/idexists/{id}": "v1",
+    "auth/keys/code":     "v1",
+    "auth/keys":          "v1",
+    "sql":                         "v1",
+    "sql/ddl":                     "v1",
+    "sql/dml":                     "v1",
+    "sql/dql":                     "v1",
+    "encryption/sql/dql":          "v1",
+    "encryption/sql/dml":          "v1",
+    "encryption/configure":        "v1",
+    "sql/queries/{queryName}":     "v2",
+    "sql/queries-by-id/{queryId}": "v2",
+    "discover/namespace":                     "v1",
+    "discover/schema":                        "v2",
+    "discover/table":                         "v1", 
```

Comment:
It looks like we're using V1 for some but not all of the Discovery APIs. Can we make sure all the discover APIs are using V2? The planned removal of V1 APIs is slated for EOY